### DISCUSSION TITLE
Minimal amount of training samples for new NER model

### DISCUSSION THREAD

**Discussion by dave-espinosa** [explosion/spaCy#10752] [Help: Best practices]
Hello everyone,

Using [this flowchart](https://support.prodi.gy/t/annotation-flowchart-named-entity-recognition/1556)  for my project, it seems I must "Train a new model from scratch", using `ner.manual` (please take my word for granted, there is no need in digging in details 😁). However and if my understanding of that flowchart is correct:
- It looks like I need at least 4000 labeled texts, to train my model, is that right?
- If after training my model, results still don't look fine, another 4000 samples would be needed, is that correct?
- In general, are Prodigy recommendations in that flowchart "the same" than the ones suggested to build a NER model in spaCy?

Additionally, no recommendation regarding Catastrophic Forgetting is included in that flowchart, why? (PS.: I know there are already some posts in this forum, just curious as why such best practice has not been included there).

Thank you!

**Comment by polm** [comment]
The number of examples required for a model are a guideline, so the exact numbers aren't really the important thing. Basically if you find performance isn't good, the first thing you should focus on is more data.

Catastrophic forgetting isn't inherent to NER, it's only an issue if you try to re-train a model with different data. You can avoid it by just training a model from scratch or using all your data. I suspect many people never encounter it at all.
